src/main/java/org/opentripplanner/street/search/state/StateData.java

```diff
@@ -3,7 +3,7 @@
 import java.util.ArrayList;
 import java.util.List;
 import org.opentripplanner.routing.api.request.StreetMode;
-import org.opentripplanner.routing.vehicle_rental.RentalVehicleType.FormFactor;
+import org.opentripplanner.service.vehiclerental.model.RentalVehicleType.FormFactor;
```

Comment:
I'd say we pull off `org.opentripplanner.service.vehiclerental.model.RentalVehicleType.FormFactor` and move it inside the street model in order to get rid of this circular dependency.